lib/storage/data/external/PfsClient.js

```diff
@@ -36,18 +36,6 @@ class PfsClient {
 
     put(stream, size, keyContext, reqUids, callback) {
         const log = createLogger(reqUids);
-        if (keyContext.metaHeaders['x-amz-meta-mdonly'] === 'true'
```

Comment:
The reason why I left this here is to deal with the delete markers and to handle 0-byte objects. This is because the code I added in cloudserver can't differentiate between a 0-byte PUT and an MD only put.

You can see in my original PR to deprecate this code:
https://github.com/scality/Arsenal/pull/836#issuecomment-517067466